versioned_docs/version-v2.16.x/extend/extend-apiml/api-mediation-oidc-authentication.md

```diff
@@ -44,10 +44,13 @@ The following diagram illustrates the interactions between the participants of t
 - The user is asked to provide valid credentials (authentication factors).
 - After successful validation of all authentication factors, the OIDC provider grants the client an Access Token.
 - The client can then request from API ML Gateway the needed mainframe resources presenting the access token in the request.
-- The Gateway validates the access token by comparing the key id of the token against the key ids obtained from the authorization server's JWK keys endpoint.
+- The Gateway validates the access token in two ways:
+    -  By comparing the key ID of the token against the key ids obtained from the authorization server's JWK keys endpoint (`oidc.validationType: JWK` or omited configuration (`oidc.validationType`)).
```

Comment:
Second use of "ID" still needs to be capitalized: "... the key **IDs** obtained from ..."